releasing/cloudbuild_kustomize_image.yaml

```diff
@@ -29,6 +29,8 @@ steps:
         COMMIT=$(git rev-parse HEAD)
         --build-arg
         DATE=$(date -u +%FT%TZ)
+        --platform
```

Comment:
Looks like there is no `--platform` option for `docker build`.
Are you trying to use the `buildx` command?

```
kobdotsh@instance-2:~$ sudo docker run -it --entrypoint=/bin/bash --rm gcr.io/cloud-builders/docker
root@9313da0b0c4d:/# docker build --help

Usage:  docker build [OPTIONS] PATH | URL | -

Build an image from a Dockerfile

Options:
      --add-host list           Add a custom host-to-IP mapping (host:ip)
      --build-arg list          Set build-time variables
      --cache-from strings      Images to consider as cache sources
      --cgroup-parent string    Optional parent cgroup for the container
      --compress                Compress the build context using gzip
      --cpu-period int          Limit the CPU CFS (Completely Fair Scheduler)
                                period
      --cpu-quota int           Limit the CPU CFS (Completely Fair Scheduler)
                                quota
  -c, --cpu-shares int          CPU shares (relative weight)
      --cpuset-cpus string      CPUs in which to allow execution (0-3, 0,1)
      --cpuset-mems string      MEMs in which to allow execution (0-3, 0,1)
      --disable-content-trust   Skip image verification (default true)
  -f, --file string             Name of the Dockerfile (Default is
                                'PATH/Dockerfile')
      --force-rm                Always remove intermediate containers
      --iidfile string          Write the image ID to the file
      --isolation string        Container isolation technology
      --label list              Set metadata for an image
  -m, --memory bytes            Memory limit
      --memory-swap bytes       Swap limit equal to memory plus swap: '-1' to
                                enable unlimited swap
      --network string          Set the networking mode for the RUN
                                instructions during build (default "default")
      --no-cache                Do not use cache when building the image
      --pull                    Always attempt to pull a newer version of the
                                image
  -q, --quiet                   Suppress the build output and print image ID
                                on success
      --rm                      Remove intermediate containers after a
                                successful build (default true)
      --security-opt strings    Security options
      --shm-size bytes          Size of /dev/shm
      --squash                  Squash newly built layers into a single new layer
  -t, --tag list                Name and optionally a tag in the 'name:tag' format
      --target string           Set the target build stage to build.
      --ulimit ulimit           Ulimit options (default [])

Reply:
@koba1t good catch! newer versions of docker has `build` automatically use `buildx build`, looks like the GCR image has an older version. I don't have a cloud build environment to test this, would you be able to verify this works? Thanks!